server/SchemaLoader.js

```diff
@@ -0,0 +1,127 @@
+"use strict";
+
+const EventEmitter = require("events");
+const { remote } = require("electron");
+const jsonfile = require("jsonfile");
+const mkdirp = require("mkdirp");
+const path = require("path");
+const fs = require("fs");
+
+const ValidationSchema = require("./model/ValidationSchema");
+
+class SchemaLoader extends EventEmitter {
+    constructor() {
+        super();
```

Comment:
Isn't this call implied (and so can be left out)?

Reply:
Nope. When extending this must be called before any reference to `this`

Reply:
Interesting case where JS is more verbose than Java :thinking: